lib/seal.rb

```diff
@@ -26,7 +26,7 @@ def bark_at(team, mode: nil)
     begin
       message = case mode
                 when "quotes"
-                  Message.new(team.quotes.sample)
+                  Message.new(team.quotes.sample) if team.quotes_days.map(&:downcase).include?(Date.today.strftime("%A").downcase)
```

Comment:
Nice that it accounts for downcase "monday" etc, but I don't see an associated test for this? You could change just one of the 'lions' team's quotes_days to lowercase and write a unit test to cover that.